website/pages/docs/nia/index.mdx

```diff
@@ -0,0 +1,33 @@
+---
+layout: docs
+page_title: Network Infrastructure Automation
+sidebar_title: Network Infrastructure Automation
+description: >-
+  Network Infrastructure Automation (NIA) relies on a declarative, workflow and service driven network automation architecture. Consul-Terraform-Sync is a daemon based architecture that utilizes Terraform as the underlying automation tool and leverages the Terraform provider ecosystem to drive relevant change to the network infrastructure. This usage of the Terraform provider in conjunction with Consul allows for Consul to act as the core data source to inform updates to the state of the infrastructure.
+---
+
+# Network Infrastructure Automation
+
+Network Infrastructure Automation (NIA) enables dynamic updates to network infrastructure devices triggered by service changes. Consul-Terraform-Sync utilizes Consul as a data source that contains networking information about services and monitors those services. Terraform is used as the underlying automation tool and leverages the Terraform provider ecosystem to drive relevant change to the network infrastructure.
+
+Consul-Terraform-Sync executes one or more automation tasks with the most recent service variable values from the Consul service catalog. Each task consists of a runbook automation written as a Consul-Terraform-Sync compatible Terraform module using resources and data sources for the underlying network infrastructure. The “Consul-Terraform-Sync” daemon runs on the same node as a Consul agent.
```

Comment:
It seems like we're using lower-case `consul-terraform-sync` to refer to the daemon and upper-case to refer to the product? if so, maybe "“Consul-Terraform-Sync” daemon" => "consul-terraform-sync daemon"

Reply:
I was using it interchangeably. That's a good idea, we can use all lower case for the daemon and the first letters upper case for the product. Thank you!

Reply:
Not a big issue, but just a bit of CLI/Unix nomenclature. It is still usually referred to as a binary, program, executable, etc. The term daemon is more used to reference how it runs in the background.